frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -247,12 +247,14 @@ string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text
 
 string.GitMachete.EnhancedGraphTable.automatic-discover.notification.title.cannot-discover-layout-error=Can't automatically discover repository layout
 string.GitMachete.EnhancedGraphTable.automatic-discover.task-title=Automatically discovering branch layout
+action.GitMachete.EnhancedGraphTable.automatic-discover.slide-out-skipped=Slide out skipped branches
```

Comment:
Missing `?` ?

Reply:
I do not recall the question form of the notification actions. One can see `View commits` (not `View commits?`). IMHO that would be a bit inconsistent and strange but I can change it if you want.

Reply:
Right, ignore